src/components/__test__/Component.tes.js

```diff
@@ -0,0 +1,4 @@
+// import { render, screen } from '@testing-library/react';
+// //
+
+// test('renders the home page', () => {});
```

Comment:
<img width="919" alt="image" src="https://github.com/Jeddaa/covidata/assets/65068771/253b053c-a41e-4941-9250-3de834a8ec9c">

- [ ] Nice work so far your app is working well however  yous till need tp work on the design of you project to look like the template. kindly make sure the colloer choosen is on of the template one, the design should look exaxtly like the  [template design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).

Your design | the template
---|---
<img width="256" alt="image" src="https://github.com/Jeddaa/covidata/assets/65068771/9c6eb6af-3bfc-4e88-b5bf-c918cd3943ad"> | <img width="260" alt="image" src="https://github.com/Jeddaa/covidata/assets/65068771/b52831be-7453-4ad5-92b5-87fde9b59f1f"> <img width="177" alt="image" src="https://github.com/Jeddaa/covidata/assets/65068771/8989e9d0-d7df-4022-8b64-4a05c35c8f6f">